### ISSUE TITLE
Unit test in test_Polynomial.praat fails randomly

### ISSUE THREAD

**Issue by rlaboiss** [praat/praat#492]
One of the unit tests in file dwtest/test_Polynomial.praat fails randomly:

```
$ while true ; do xvfb-run -a praat --run dwtest/test_Polynomial.praat ; done
[…]
test_Polynomial
...Roots
Error: Script assertion fails in line 46 (false):
   .dif < .eps2 * abs (.roots[.j]); -0.872491868914407 -0.8724892132017376 .j
Script “/var/data/rlaboiss/debian/PKGS/praat/praat/dwtest/test_Polynomial.praat” not completed.
Praat: script command <<dwtest/test_Polynomial.praat>> not completed.
```

I think that increasing the value of `.eps2` to 1e-4 may help to avoid the problem, but even that high tolerance value is not a guaranty that the test will always succeed.

Would it be possible to find a more principled way to make this test succeeding in a deterministic way?

**Comment by davidweenink** [comment]
Well may be. I rewrote the test to not fail on this issue. 
Polynomials can vary wildly and I don't know anything better for the moment.

**Comment by rlaboiss** [comment]
Thanks, David. Just for the record, this is the commit that fixed the problem: bc4d9026a0296676ab557715faa2af2b58ceb9f7
